src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -664,7 +664,7 @@ public LinearLayout.Alignment ItemAlignment
         /// The value is applied when there exist icon and text both.
```

Comment:
```suggestion
        /// The value is applied when the button has both icon visual and text set.
```
I'm not sure about "icon visual". You can drop the word "visual".
BTW - how user sets the "icon"? There are properties for various visuals. Does one of them relate to icon?

Reply:
Done.
User can set icon using `iconURL` property.